test/buildAndRun.test.ts

```diff
@@ -82,14 +81,15 @@ suite("Build Image", function (this: Suite): void {
             errorHandling: {}
         };
 
-        await configure(context, testRootFolder);
-        assert.equal(configureInputs.length, 0, 'Not all inputs were used for configure docker files');
+        await testUserInput.runWithInputs(configureInputs, async () => {
+            await configure(context, testRootFolder);
```

Comment:
Just to make sure I understand, but you got rid of these asserts since checking whether or not all inputs were used should be handled in testUserInput anyway, right?

Reply:
Yes it's handled in the shared code now